ow_lander/src/ow_lander/actions.py

```diff
@@ -1139,7 +1136,7 @@ def _is_position_in_sample_dock(self, position):
     # NOTE: This value can be found by adding the y-value on line 25 of
```

Comment:
Line numbers can change. It would be better to describe where the values are coming from with link or joint names.

Reply:
Done